homeassistant/components/hassio/backup.py

```diff
@@ -19,39 +24,77 @@
     NewBackup,
     WrittenBackup,
 )
-from homeassistant.core import HomeAssistant
+from homeassistant.core import HomeAssistant, callback
 from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
 
+from .const import DOMAIN, EVENT_SUPERVISOR_EVENT
 from .handler import get_supervisor_client
 
+LOCATION_CLOUD_BACKUP = ".cloud_backup"
+
 
 async def async_get_backup_agents(
     hass: HomeAssistant,
     **kwargs: Any,
 ) -> list[BackupAgent]:
     """Return the hassio backup agents."""
-    return [SupervisorLocalBackupAgent(hass)]
+    client = get_supervisor_client(hass)
+    mounts = await client.mounts.info()
```

Comment:
We need to handle `SupervisorError` for all the client calls. We can add it later when I've added a manager error to raise from.